$MirOS: ports/devel/glib/patches/patch-glib_gbookmarkfile_c,v 1.1 2008/03/06 22:08:24 tg Exp $
--- glib/gbookmarkfile.c.orig	Fri Sep 12 19:55:28 2008
+++ glib/gbookmarkfile.c	Thu Nov 25 12:24:44 2010
@@ -263,7 +263,7 @@ bookmark_app_info_dump (BookmarkAppInfo 
                             BOOKMARK_APPLICATION_ELEMENT,
                             BOOKMARK_NAME_ATTRIBUTE, name,
                             BOOKMARK_EXEC_ATTRIBUTE, exec,
-                            BOOKMARK_TIMESTAMP_ATTRIBUTE, (time_t) app_info->stamp,
+                            BOOKMARK_TIMESTAMP_ATTRIBUTE, (long) app_info->stamp,
                             BOOKMARK_COUNT_ATTRIBUTE, app_info->count);
 
   g_free (name);
